e2e/tests/controls/rendering/index.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, test } from '@playwright/test';
+import { CanvasUtil } from '../../../utils/CanvasUtil';
+import data from './data.json';
+import data2 from './data2.json';
+
+import '../../../setup';
+
+test.describe('control box rendering', () => {
+  [
+    { data, title: 'skewY, flipX' },
+    { data: data2, title: 'skewY' },
+  ].forEach(({ data, title }) => {
+    [0, 10].forEach((padding) => {
+      test(`${title} and padding of ${padding}`, async ({ page }) => {
+        const canvasUtil = new CanvasUtil(page);
+        await canvasUtil.executeInBrowser(
+          async (canvas, [data, padding]) => {
+            await canvas.loadFromJSON(data);
+            canvas.skipOffscreen = false;
+            canvas.renderAll();
+            canvas.getObjects().forEach((object) => {
+              object.padding = padding;
+              object.borderScaleFactor = 3;
+              object.transparentCorners = false;
+              const color = object.fill;
+              object._renderControls(canvas.contextContainer, {
+                borderColor: color,
+                cornerColor: color,
+              });
+              object.getObjects?.().forEach((subTarget) => {
+                subTarget.padding = padding;
```

Comment:
added this line

Reply:
the rest is unchanged

Reply:
can you do 2 * padding or 3 * padding in this subTarget? i want to see the  controls inside the group getting out of the group with a too large padding. This will ensure we cover weird situations we could break the code to

Reply:
done